### PR TITLE
feat: page metadata

### PR DESCRIPTION
**Pull request by dzbo**
### Ticket ID

[DEV-6011](https://app.clickup.com/t/2645698/DEV-6011)

### Description

Add page metadata.

### CODE REVIEW

**Review by Hugoo — APPROVED** [review]
maybe here we can have something more "social media ready" with text in it, like for our docs:

image

or like this:

image

File: components/websiteMetadata.vue [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<script setup lang="ts">
+const { formatMessage } = useIntl()
+const title = ref<string>()
+const description = ref<string>()
+const domain = ref('wallet.universalprofile.cloud')
```
Comment (richtera) [inline components/websiteMetadata.vue:5]: I think this should be `document.location.hostname`. But I think the metadata headers won't work unless it's SSR. Hmmm, what's the use case for this data?

**Review by richtera — COMMENTED** [review]
I don't think these metadata fields will work when dynamically initialized.

File: components/websiteMetadata.vue (hunk 1 above)
Comment (dzbo) [reply components/websiteMetadata.vue:5]: I will check with `document.location.hostname`. This is for adding proper `meta` tags to the page. We don't use SSR for this project.

**Comment by dzbo** [comment]
> maybe here we can have something more "social media ready" with text in it, like for our docs:

yeah, maybe flag this on channel, this is more feedback to Tim.

File: components/websiteMetadata.vue (hunk 1 above)
Comment (richtera) [reply components/websiteMetadata.vue:5]: I See. I think Twitter metadata will only work with SSR. Need to debug

Comment (dzbo) [reply components/websiteMetadata.vue:5]: Alright, alternative is to hardcode this things.

**Pull request merged**